pasts.lv,"HTTPSConnectionPool(host='pasts.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='pasts.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
patefon.co,"HTTPSConnectionPool(host='patefon.co', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='patefon.co', port=443): Read timed out. (read timeout=2)"
patmar.ee,"HTTPSConnectionPool(host='patmar.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='patmar.ee', port=443): Read timed out. (read timeout=2)"
patro.cz,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
pbh.gov.br,"HTTPSConnectionPool(host='pbh.gov.br', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f366c320c40>, 'Connection to pbh.gov.br timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pbh.gov.br', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f366d84c160>, 'Connection to pbh.gov.br timed out. (connect timeout=2)'))"
pcfarm.ro,"HTTPSConnectionPool(host='pcfarm.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPSConnectionPool(host='pcfarm.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))"
pcgamer.com,"HTTPSConnectionPool(host='pcgamer.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f366bd80910>, 'Connection to pcgamer.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pcgamer.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f366ca5ec70>, 'Connection to pcgamer.com timed out. (connect timeout=2)'))"
pclab.pl,"HTTPSConnectionPool(host='pclab.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='pclab.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
pctown.co.nz,"HTTPSConnectionPool(host='pctown.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='pctown.co.nz', port=443): Read timed out. (read timeout=2)"
pdn.ac.lk,"HTTPSConnectionPool(host='pdn.ac.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366c1bc370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pdn.ac.lk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366ddad550>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
peatus.ee,"HTTPSConnectionPool(host='web.peatus.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='web.peatus.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pedagogica.edu.co,"HTTPSConnectionPool(host='pedagogica.edu.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366a8c5f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pedagogica.edu.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366c15f760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pediatriepropraxi.cz,"HTTPSConnectionPool(host='pediatriepropraxi.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='pediatriepropraxi.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
pelando.com.br,"HTTPSConnectionPool(host='www.pelando.com.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pelando.com.br', port=443): Read timed out. (read timeout=2)"
penang.gov.my,"HTTPSConnectionPool(host='penang.gov.my', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366d32a820>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='penang.gov.my', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366b96bfd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
peoples.ru,"HTTPSConnectionPool(host='peoples.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.peoples.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
peoplesbank.lk,"HTTPSConnectionPool(host='peoplesbank.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366c2ec2e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='peoplesbank.lk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366a23ef40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
perach.org.il,"HTTPSConnectionPool(host='perach.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='perach.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
perak.gov.my,"HTTPSConnectionPool(host='perak.gov.my', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366a0afc40>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='perak.gov.my', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366a1e8d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
peritiuniti.it,"HTTPSConnectionPool(host='peritiuniti.it', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='peritiuniti.it', port=80): Read timed out. (read timeout=2)"
persee.fr,"HTTPSConnectionPool(host='www.persee.fr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.persee.fr', port=443): Read timed out. (read timeout=2)"
petah-tikva.muni.il,"HTTPSConnectionPool(host='petah-tikva.muni.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f366e716700>, 'Connection to petah-tikva.muni.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='petah-tikva.muni.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f366c99c430>, 'Connection to petah-tikva.muni.il timed out. (connect timeout=2)'))"
petitlyrics.com,"HTTPSConnectionPool(host='petitlyrics.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='petitlyrics.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
petlove.com.br,"HTTPSConnectionPool(host='www.petlove.com.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.petlove.com.br', port=443): Read timed out. (read timeout=2)"
pfister.ch,"HTTPSConnectionPool(host='www.pfister.ch', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pfister.ch', port=443): Read timed out. (read timeout=2)"
pgdlisboa.pt,"HTTPSConnectionPool(host='pgdlisboa.pt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f366cb74760>, 'Connection to pgdlisboa.pt timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pgdlisboa.pt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3667c08040>, 'Connection to pgdlisboa.pt timed out. (connect timeout=2)'))"
pgr.gob.ni,"HTTPSConnectionPool(host='pgr.gob.ni', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3668d234f0>: Failed to establish a new connection: [Errno 113] No route to host'))","HTTPConnectionPool(host='pgr.gob.ni', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366dac1fa0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
pharm16.gr,"HTTPSConnectionPool(host='www.pharm16.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pharm16.gr', port=443): Read timed out. (read timeout=2)"
pharmaline.co.il,"HTTPSConnectionPool(host='pharmaline.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f366e857a30>, 'Connection to pharmaline.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pharmaline.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f366c274e50>, 'Connection to pharmaline.co.il timed out. (connect timeout=2)'))"
pharmilion.gr,"HTTPSConnectionPool(host='pharmilion.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='pharmilion.gr', port=443): Read timed out. (read timeout=2)"
phimmoizz.net,"HTTPSConnectionPool(host='phimmoizz.net', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='phimmoizz.net', port=80): Read timed out. (read timeout=2)"
phunuonline.com.vn,"HTTPSConnectionPool(host='phunuonline.com.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3666736640>, 'Connection to phunuonline.com.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='phunuonline.com.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3666851610>, 'Connection to phunuonline.com.vn timed out. (connect timeout=2)'))"
pichinchacomunicaciones.com.ec,"HTTPSConnectionPool(host='pichinchacomunicaciones.com.ec', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f3666e73910>, 'Connection to pichinchacomunicaciones.com.ec timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pichinchacomunicaciones.com.ec', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3666409460>, 'Connection to pichinchacomunicaciones.com.ec timed out. (connect timeout=2)'))"
pikabu.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
pikiwiki.org.il,"HTTPSConnectionPool(host='www.pikiwiki.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pikiwiki.org.il', port=443): Read timed out. (read timeout=2)"
pikk.ee,"HTTPSConnectionPool(host='www.pikk.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pikk.ee', port=443): Read timed out. (read timeout=2)"
piluli.ru,"HTTPSConnectionPool(host='piluli.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='piluli.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pirkka.fi,"HTTPSConnectionPool(host='pirkka.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.pirkka.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pitat.com,"HTTPSConnectionPool(host='pitat.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f36696735e0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='pitat.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366ddf0130>: Failed to establish a new connection: [Errno 111] Connection refused'))"
pivnidenicek.cz,"HTTPSConnectionPool(host='pivnidenicek.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pivnidenicek.cz' doesn't match 'www.pivnidenicek.cz'"")))","HTTPSConnectionPool(host='pivnidenicek.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pivnidenicek.cz' doesn't match 'www.pivnidenicek.cz'"")))"
pizzasnella.it,"HTTPSConnectionPool(host='pizzasnella.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366f3435e0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='www.pizzasnella.it', port=80): Read timed out. (read timeout=2)"
planalto.gov.br,"HTTPSConnectionPool(host='planalto.gov.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1124)')))","(""Connection broken: ConnectionResetError(104, 'Connection reset by peer')"", ConnectionResetError(104, 'Connection reset by peer'))"
pleasantgroveumc.net,"HTTPSConnectionPool(host='pleasantgroveumc.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3668a9cd30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pleasantgroveumc.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366c890760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
plovdiv.bg,"HTTPSConnectionPool(host='plovdiv.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='plovdiv.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
plz.lv,"HTTPSConnectionPool(host='www.plz.lv', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.plz.lv', port=443): Read timed out. (read timeout=2)"
pm.gov.jo,"HTTPConnectionPool(host='pm.gov.jo', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='pm.gov.jo', port=80): Read timed out. (read timeout=2)"
pmda.go.jp,"HTTPSConnectionPool(host='pmda.go.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f366e64c730>, 'Connection to pmda.go.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pmda.go.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3665d34340>, 'Connection to pmda.go.jp timed out. (connect timeout=2)'))"
pmnewsnigeria.com,"HTTPSConnectionPool(host='www.pmnewsnigeria.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pmnewsnigeria.com', port=443): Read timed out. (read timeout=2)"
pnc.gob.sv,"HTTPSConnectionPool(host='pnc.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366809b940>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pnc.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f36667c4b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pnp.co.za,"HTTPSConnectionPool(host='pnp.co.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f36652e3dc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pnp.co.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f36679a4f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
podcastid.ee,"HTTPSConnectionPool(host='podcastid.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='podcastid.ee', port=443): Read timed out. (read timeout=2)"
poder-judicial.go.cr,"HTTPSConnectionPool(host='poder-judicial.go.cr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f366f2442e0>, 'Connection to poder-judicial.go.cr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poder-judicial.go.cr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3665f16160>, 'Connection to poder-judicial.go.cr timed out. (connect timeout=2)'))"
poderjudicial.gob.ni,"HTTPSConnectionPool(host='poderjudicial.gob.ni', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f366e18c820>, 'Connection to poderjudicial.gob.ni timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poderjudicial.gob.ni', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f3665303f40>, 'Connection to poderjudicial.gob.ni timed out. (connect timeout=2)'))"
poduckun.net,"HTTPSConnectionPool(host='poduckun.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='poduckun.net', port=443): Read timed out. (read timeout=2)"
poipeplekkers.be,"HTTPSConnectionPool(host='poipeplekkers.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3670712730>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='poipeplekkers.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366c3664c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
poisci.me,"HTTPSConnectionPool(host='poisci.me', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='poisci.me', port=443): Read timed out. (read timeout=2)"
polazak.rs,Exceeded 30 redirects.,Exceeded 30 redirects.
policia.gov.co,"HTTPSConnectionPool(host='policia.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f36622511c0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='policia.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f36695f4970>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
politikaplus.com,"HTTPSConnectionPool(host='politikaplus.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='politikaplus.com', port=80): Read timed out. (read timeout=2)"
polito.it,"HTTPSConnectionPool(host='polito.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f36681c11f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='polito.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366206cfd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
polsl.pl,"HTTPSConnectionPool(host='polsl.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3661fb9070>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='polsl.pl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f3665836760>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
pom.go.id,"HTTPSConnectionPool(host='pom.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='pom.go.id', port=443): Max retries exceeded with url: /new/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pontofrio.com.br,"HTTPConnectionPool(host='www.pontofrio.com.br', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.pontofrio.com.br', port=80): Read timed out. (read timeout=2)"
popcornfor2.com,"HTTPSConnectionPool(host='popcornfor2.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='popcornfor2.com', port=80): Read timed out. (read timeout=2)"
popvortex.com,"HTTPSConnectionPool(host='popvortex.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'popvortex.com' doesn't match either of '*.gridserver.com', 'gridserver.com'"")))","HTTPConnectionPool(host='www.popvortex.com', port=80): Read timed out. (read timeout=2)"
poradnikogrodniczy.pl,"HTTPSConnectionPool(host='poradnikogrodniczy.pl', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f36708d1520>, 'Connection to poradnikogrodniczy.pl timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poradnikogrodniczy.pl', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f366a4b14f0>, 'Connection to poradnikogrodniczy.pl timed out. (connect timeout=2)'))"
porezna-uprava.hr,"HTTPSConnectionPool(host='porezna-uprava.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366f0419d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='porezna-uprava.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366e127910>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
port2port.co.il,"HTTPSConnectionPool(host='port2port.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.port2port.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
portal.gov.bd,"HTTPSConnectionPool(host='portal.gov.bd', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='portal.gov.bd', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
portaldasfinancas.gov.pt,"HTTPSConnectionPool(host='portaldasfinancas.gov.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366c9afc10>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='portaldasfinancas.gov.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366c3befa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
portorecanaticalcio.it,"HTTPSConnectionPool(host='www.portorecanaticalcio.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.portorecanaticalcio.it', port=443): Read timed out. (read timeout=2)"
posted.co.rs,"HTTPSConnectionPool(host='posted.co.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f366c274d60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='posted.co.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f366a5932e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pozitivke.net,"HTTPSConnectionPool(host='pozitivke.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pozitivke.net' doesn't match either of '*.mojhost.net', 'mojhost.net'"")))","HTTPConnectionPool(host='pozitivke.net', port=80): Read timed out. (read timeout=2)"
